server/db_mysql.go

```diff
@@ -269,6 +304,20 @@ func (ms *MsqlCache) UpdateFixity(item string, status string, notes string) erro
 	return err
 }
 
+func (ms *MsqlCache) DeleteFixity(id string) error {
```

Comment:
minor name suggestion, maybe `DeleteFixityRecord`?

Should this method be doing a check that the record is "scheduled"? That could be handled in the exec `DELETE FROM fixity WHERE id = ? AND state = "scheduled"`